tests/Http/HttpRequestTest.php

```diff
@@ -122,7 +122,7 @@ public function testFullUrlMethod()
         $this->assertEquals('https://foo.com', $request->fullUrl());
 
         $request = Request::create('https://foo.com', 'GET');
-        $this->assertEquals('https://foo.com?coupon=foo', $request->fullUrlWithQuery(['coupon' => 'foo']));
+        $this->assertEquals('https://foo.com/?coupon=foo', $request->fullUrlWithQuery(['coupon' => 'foo']));
```

Comment:
This is fixing a wrong test that was incorrectly put here. Check lines 127-134 which is the correct behaviour not put by me. Notice how they add the slash? Why the inconsistency? Slash must be added if there is no path but query is present.